backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -2127,8 +2414,41 @@ describe('ContributionResolver', () => {
                       amount: '500',
                       moderator: admin.id,
                       creation: ['1000', '800', '500'],
-                    },
-                    {
+                    }),
+                    expect.not.objectContaining({
```

Comment:
No not, test for the length

Reply:
I will need to go on this tests on the next PR 'refactor listUncofirmedContributions to adminListAllContributions', would it be OK to let this slide until then?